facts/test_inventory.py

```diff
@@ -160,7 +160,7 @@ def test_bitmasktonetmask():
 def test_genvlans():
     '''test cases for the genvlans() function'''
     cases = [
-        ["VVRNG\ttest_vlan_\t\t200-201\t2001:470:f325:200::/54\t10.2.0.0/15\tdynamic vlan", [
+        ["VVRNG\ttest_vlan_\t\t200-201\t2001:470:f325::/48\t10.2.0.0/15\tdynamic vlan", [
```

Comment:
thanks for including and updating testing

Reply:
I'm slightly concerned about the use of 2001:470:f325:: throughout this as the current prefix is 2001:470:f026::/48.

Reply:
I believe the intent of this unit test is to validate *ANY* /48 and not necessarily the real one. @sarcasticadmin can probably clarify if the intent I read was different.

Reply:
@kylerisse correct, this test is a generic case and the prefix doesnt hurt the validity of the test. 

We can update it to avoid confusion and that probably would be a good first issue for someone :slightly_smiling_face: since I think there a quite a few references to the old prefix.